warehouse/pyproject.toml

```diff
@@ -25,6 +25,12 @@ dbt-bigquery = "^1.3.0"
 typer = "^0.7.0"
 gcsfs = "^2023.1.0"
 dbt-metabase = "^0.9.14"
+networkx = {version = "<3", extras = ["default"]}
+networkx-viewer = "^0.3.0"
+# https://github.com/pygraphviz/pygraphviz/issues/398#issuecomment-1450367670
+# export CFLAGS="-I $(brew --prefix graphviz)/include"
```

Comment:
what are these commented lines for? like what is the reason to leave them here but commented?

Reply:
They may need to be executed on macOS to install graphviz; they're not pyproject.toml lines. I didn't want to only include the link in case of link rot.